### PR TITLE
Models for users tasks

### PR DESCRIPTION
**Pull request by YaromyrO**
First version of userModel

### CODE REVIEW

File: src/models/userModel.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
+package models
+
+import "database/sql"
+
+var db *sql.DB
+
+type User struct {
+	Id int
```
Comment (Nastya-Kruglikova) [inline src/models/userModel.go]: We have discussed, that Id should be uuid
Id should be ID  - https://github.com/golang/go/wiki/CodeReviewComments#initialisms

File: src/models/userModel.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+func deleteUser(id int) error {
+	_, err := db.Exec("DELETE FROM user WHERE id = $1", id)
+
+	if err != nil{
+		return err
+	}
+	return nil
+}
+
+//Метод котрий вертатиме всю інформацію юзерів для Адміа
```
Comment (Nastya-Kruglikova) [inline src/models/userModel.go]: Pls use English for comments

File: src/models/userModel.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+func getUsersForAdmin() (*[]User, error) {
+
+	rows, err := db.Query("SELECT * FROM user")
+	if err != nil {
+
```
Comment (Nastya-Kruglikova) [inline src/models/userModel.go]: missing something ?

File: src/models/userModel.go [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+	}
+	return &users, nil
+}
+
+//Метод котрий вертатиме лише імя та логін юзерів
+func getUsers() (*[]User, error) {
+
+	rows, err := db.Query("SELECT name, login FROM user")
+	if err != nil {
+
```
Comment (Nastya-Kruglikova) [inline src/models/userModel.go]: missing something ?

File: src/models/userModel.go [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+	ID       uuid.UUID
+	Name     string
+	Login    string
+	Password string
+}
+
+var DB, Mock, Err = sqlmock.New()
+
+func CreateUser(user User) error {
+	_, err := DB.Exec("INSERT INTO User (name, login, password) VALUES ($1, $1, $3)", user.Name, user.Login, user.Password)
```
Comment (mshto) [inline src/models/userModel.go]: Could you replace query with const ? It'll be more readable.

Comment (mshto) [reply src/models/userModel.go]: Could you return user ID if user will be created?

**Pull request closed without merging**